font-freefont.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: font-freefont
+  version: 20120503
+  epoch: 0
+  description: A set of free high-quality TrueType fonts covering the UCS character set
+  copyright:
+    - license: GPL-3.0-or-later
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - fontconfig
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 7c85baf1bf82a1a1845d1322112bc6ca982221b484e3b3925022e25b5cae89af
+      uri: https://ftp.gnu.org/gnu/freefont/freefont-ttf-${{package.version}}.zip
+      extract: false
+
+  - runs: |
+      unzip freefont-ttf-${{package.version}}.zip
+      install -D -m644 ./freefont-${{package.version}}/*.ttf -t "${{targets.destdir}}"/usr/share/fonts/freefont
+
+update:
+  enabled: false
+  exclude-reason: >
+    they don't publish releases or tags :(
+
```

Comment:
This isn't entirely accurate. GNU has published releases, but we aren't tracking them because it's unlikely there will ever be another release